lib/rollbar/sidekiq.rb

```diff
@@ -1,36 +1,35 @@
 # encoding: utf-8
 
-PARAM_BLACKLIST = ['backtrace', 'error_backtrace', 'error_message', 'error_class']
+module Rollbar
+  PARAM_BLACKLIST = ['backtrace', 'error_backtrace', 'error_message', 'error_class']
 
-if Sidekiq::VERSION < '3'
-  module Rollbar
-    class Sidekiq
-      def call(worker, msg, queue)
-        begin
-          yield
-        rescue Exception => e
-          params = msg.reject{ |k| PARAM_BLACKLIST.include?(k) }
-          scope = { :request => { :params => params } }
+  class SidekiqErrorHandler
+    def self.handle(scope, e, params)
```

Comment:
This method should receive a `msg_or_context` and the `e` variables. Not `params` or `scope`, which are really the values `.handle` is calculating.

```ruby
def self.handle(msg_or_context, e)
  params = msg.reject{ |k| PARAM_BLACKLIST.include?(k) }
  scope = { :request => { :params => params } }

  Rollbar.scope(scope).error(e)
end
```
Does this make sense?